qa/tasks/rgw_cloudtier.py

```diff
@@ -79,7 +83,9 @@ def setup(self):
                 tier_config_params = "endpoint=" + endpoint.url() + \
                            ",access_key=" + cloud_access_key + \
                             ",secret=" + cloud_secret + \
-                            ",retain_head_object=" + cloud_retain_head_object
+                            ",retain_head_object=" + cloud_retain_head_object +\
+                            ",allow_read_through=" + cloud_allow_read_through +\
```

Comment:
Shouldn't we check if these options are actually set before adding them to tier_config_params here. Also please run cloud-transtiion test-suite too with this PR.